assets/js/current-projects.js

```diff
@@ -246,6 +268,40 @@ function cancelMobileFiltersEventHandler(e) {
     hideFiltersEventHandler(e)
     clearAllEventHandler()
 }
+//search bar event handler
+function searchEventHandler(){
```

Comment:
The [event property](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) is deprecated. You can pass the event as a parameter.
```suggestion
function searchEventHandler(e){
e.preventDefault();
```